Controllers/HomeController.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.AspNetCore.Mvc;
+using ProductsCatalog.Data;
+using ProductsCatalog.Models;
+
+namespace ProductsCatalog.Controllers{
+    [ApiController]
+    
+    public class HomeController : ControllerBase {
```

Comment:
I suggest the controller name to be the same as the API's resource name: `ProdutcsController`

Reply:
just to understand, we dont need a home controller? can we have multiple controllers in one api? if yes I understand why the name change..